### PR TITLE
Scrollfloat: dont unblock throttling on promise rejection

### PR DESCRIPTION
**Pull request by mikesherov**

### CODE REVIEW

**Comment by Aintaer** [comment]
:+1:

File: ux/scrollfloat.js [hunk 1]
```diff
@@ -75,9 +74,19 @@ define([
...
     function onHit() {
-      var retval = throttle(callback);
+      if (onHit.blocking) { return; }
+      onHit.blocking = true;
+
+      var retval = callback.apply(null, arguments);
+
       if (retval && typeof retval.then === 'function') {
-        retval.then(scrollCache[context]);
+        retval.then(function() {
```
Comment (mrjoelkemp) [inline ux/scrollfloat.js]: Shouldn't this be `finally`?

Comment (mrjoelkemp) [inline ux/scrollfloat.js]: No. We want this to stop fetching on an error.

**Comment by mrjoelkemp** [comment]
:+1:

**Pull request merged**